src/components/CompareResults/RevisionRow.tsx

```diff
@@ -265,10 +265,10 @@ const platformIcons: Record<PlatformShortName, ReactNode> = {
   Unspecified: '',
 };
 
-const confidenceIcons = {
-  Low: <KeyboardArrowDownIcon sx={{ color: '#D7264C' }} />,
-  Medium: <DragHandleIcon sx={{ color: '#5B5B66' }} />,
-  High: <KeyboardArrowUpIcon sx={{ color: '#017A40' }} />,
+const confidenceIcon = {
```

Comment:
just wondering why you rename it here, but not in the Subtests page :) I don't mind which one you use (with s or without s) but let's try to use the same one.

Reply:
Ah, missed that, there was a `getConfidenceIcon` function at some point, but changed my mind.